app/src/main/java/com/xmartlabs/scasas/geoquiz/ui/QuizActivity.java

```diff
@@ -17,12 +25,19 @@
 public class QuizActivity extends AppCompatActivity {
   private Button falseButton;
   private Button trueButon;
+  private ImageButton nextButton;
+  private ImageButton prevButton;
+  private TextView questionTextView;
+  @NonNull
+  private List<Question> questionList = new ArrayList<>();
```

Comment:
it's ok, but in the future use `questions`. [`If you need to name a List of activities for example, just name it activities`]
(https://github.com/xmartlabs/Android-Style-Guide?files=1#field-method-variable-and-parameter-names)